powershell/public/maester/entra/Test-MtDeviceRegistrationMfaConflict.md

```diff
@@ -4,15 +4,15 @@ When both are enabled, the conditional access policy with the "Register device"
 
 #### Remediation action:
 
-When a Conditional Access policy is configured with the **Register or join devices user action**, you must set **Entra ID** > **Devices** > **Overview** > **[Entra - Device Settings](https://entra.microsoft.com/#view/Microsoft_AAD_Devices/DevicesMenuBlade/~/DeviceSettings/menuId/Devices)** - **Require Multifactor Authentication to register or join devices with Microsoft Entra** to **No**.
+When a Conditional Access policy is configured with the **Register or join devices user action** you must disable tenant-wide multifactor requirement for device registration.
 
-Otherwise, Conditional Access policies with this user action aren't properly enforced.
```

Comment:
I think this statement provides helpful context to the recommendation. Instead of a bullet, it might flow nicely to just have this sentence follow your simplified description. i.e., 
> When a Conditional Access policy is configured with the **Register or join devices user action** you must disable tenant-wide multifactor requirement for device registration. Otherwise, Conditional Access policies with this user action are not properly enforced.